finix/model/finix_list.py

```diff
@@ -0,0 +1,171 @@
+from typing import List
+
+
+# finix list implementation
+class FinixList:
+    def __init__(self, some_list, method, **kwargs):
+        self._method = method
+        self._kwargs = kwargs
+
+        if hasattr(some_list, 'embedded'):
+            tmp = some_list.embedded
+            if isinstance(tmp, dict):
+                for key in tmp:
+                    self._contents = tmp[key]
+                    break
+            else:
+                self._contents = []
+        else:
+            self._contents = []
+        
+        self.content = []
+        if self._contents != []:
+            for item in self._contents:
+                finix_item = FinixObject(item)
+                self.content.append(finix_item)
+
+        if hasattr(some_list, 'links'):
+            self.links = some_list.links
+        else:
+            self.links = 'no_links_received'
+
+        if hasattr(some_list, 'page'):
+            self.page = some_list.page
+        else:
+            self.page = 'no_page_received'
+    
+        self.has_more = False
+        self._next_case = 0
+        if 'next_cursor' in self.page and self.content != []:
+            if self.page['next_cursor'] != 'Null' and self.page['next_cursor'] != 'null':
+                self.has_more = True
+                self._next_case = 1
+        
+        if 'offset' in self.page and self.content != []:
```

Comment:
`offset` and `next_cursor` seem to be mutually exclusive. Is that enforced anywhere? Otherwise, it looks like `offset` will take precedence here.

Reply:
It's enforced in our APIs. Old APIs use 'offset' and there is no cursor related info, new APIs use 'next_cursor' and there is no 'offset'